docs/content/_index.md

```diff
@@ -10,16 +10,33 @@ draft: false
 
 ## What is Guardicore Infection Monkey?
 
-The Infection Monkey is an open source security tool for testing a data center's resiliency to perimeter breaches and internal server infection. The Monkey uses various methods to self propagate across a data center and reports success to a centralized Monkey Island Command and Control server.
+The Infection Monkey is an open-source breach and attack simulation tool for testing a data center's resiliency to perimeter breaches and internal server infection.
+Infection Monkey will help you validate existing security solutions and will provide a view of the internal your network from an attacker's perspective.
+
+Infection Monkey is free and can be downloaded from [our homepage](https://infectionmonkey.com/).
 
 ![Infection Monkey Documentation Hub Logo](/images/monkey-teacher.svg?height=400px "Infection Monkey Documentation Hub Logo")
 
-The Infection Monkey is comprised of two parts:
+## How it works
+
+Architecturally, Infection Monkey is comprised of two parts:
+
+* Monkey Agent (Monkey for short) - a safe, worm-like binary program which scans, propagates and simulates attack techniques on the **local network**.
+* Monkey Island Server (Island for short) - a C&C web server which provides a GUI for users and interacts with the Monkey Agents.
+
+User runs Monkey Agent on the Island server machine or distributes Monkey Agent binaries on the network manually. Based on 
+the configuration parameters, Monkey Agents scan, propagate and simulate attackers behavior on the local network. All of the 
```

Comment:
```suggestion
the configuration parameters, Monkey Agents scan, propagate and simulate an attacker's behavior on the local network. All of the 
```

Reply:
This has to be `an attacker's` or `attackers'`, but `attacker's` is incorrect.

Reply:
Fixed